qiskit/transpiler/passes/routing/algorithms/token_swapper.py

```diff
@@ -54,17 +54,17 @@ def __init__(self, graph: rx.PyGraph,
         """Construct an ApproximateTokenSwapping object.
 
         Args:
-            graph (nx.Graph): Undirected graph represented a coupling map.
-            seed (Union[int, np.random.default_rng]): Seed to use for random trials.
+            graph: Undirected graph represented a coupling map.
+            seed: Seed to use for random trials.
         """
         self.graph = graph
-        self.shortest_paths = rx.graph_floyd_warshall_numpy(graph)
+        self.shortest_paths = rx.graph_distance_matrix(graph)
         if isinstance(seed, np.random.Generator):
             self.seed = seed
         else:
             self.seed = np.random.default_rng(seed)
 
-    def distance(self, vertex0: _V, vertex1: _V) -> int:
+    def distance(self, vertex0: int, vertex1: int) -> int:
```

Comment:
Is it now possible to remove the `_V` definition at the top of the file? What about `_T`?

Reply:
I'm not sure, but mostly because the python typing is still foreign to me. The the only thing still using `_V` now is the class constructor:

```python3
class ApproximateTokenSwapper(Generic[_V]):
```

which I wasn't sure what it did, so I left in place. As for `_T` yeah I missed that, since I didn't touch the one function that used it, I'll remove that now.

Reply:
Removed `_T` in: https://github.com/Qiskit/qiskit-terra/pull/5471/commits/566f65e368755d69734844127d6d7181ea662e88

Reply:
Then _V can be removed as well as 'Generic[_V]'

Reply:
Ok cool, done in: https://github.com/Qiskit/qiskit-terra/pull/5471/commits/32bc611d90165c1cf1a3dc7daf464680829748d4